TF_Tooling_Web/src/Plugins/InputActionManager.ts

```diff
@@ -0,0 +1,73 @@
+import { TouchFreeInputAction } from "../TouchFreeToolingTypes";
+import { InputActionPlugin } from "./InputActionPlugin";
+
+// Class: InputActionManager
+// The manager for all <TouchFreeInputActions> to be handled and distributed. This runs the
+// received data through any <InputActionPlugins> given to it and finaly distributes the data
+// via the  <TransmitInputAction> event which should be listened to by any class hoping to make
+// use of incoming <TouchFreeInputActions>.
+export class InputActionManager extends EventTarget {
+    // Event: TransmitInputAction
+    // An event for transmitting <TouchFreeInputActions> that are received via the <messageReceiver> to
+    // be listened to.
+
+    // Event: TransmitInputActionRaw
+    // An event for immediately transmitting <TouchFreeInputActions> that are received via the
+    // <messageReceiver>to be listened to.
```

Comment:
I'd explicitly call out that this is _before_ plugins are executed as opposed to the other which is post-plugins. Also there's a missing space here between the close angle-brace and the word which may mess up NaturalDocs' generation